app/assets/javascripts/discourse/tests/acceptance/create-invite-modal-test.js

```diff
@@ -242,3 +243,18 @@ acceptance(
     });
   }
 );
+
+acceptance(
+  "Invites - Populates Edit Invite Form with saved invite data",
+  function (needs) {
+    needs.user();
+
+    test("shows correct saved data in form", async function (assert) {
+      await visit("/u/eviltrout/invited/pending");
+      await click(
+        ".user-invite-list tbody tr:nth-child(1) .invite-actions .btn:nth-child(1)"
+      ); // first invite edit button
+      assert.equal(query("#invite-max-redemptions").value, 10);
```

Comment:
🏌️ qunit-dom goodness 🙂 

```suggestion
      assert.dom("#invite-max-redemptions").hasValue(10);
```